### ISSUE TITLE
2

### ISSUE THREAD

**Issue by adenysenko** [adenysenko/q#2]

**Comment by adenysenko** [comment]
```
Context: I need to ask my boss Steven to confirm my Ukraine trip.

1)  When/how is the best to do it [1-2]
1. send him sms today
2. wait till meeting him in office tomorrow.


2) Best indicator for trading to use [1-30]
#    9    1	(JS) DMI Bars - DMI Bars
#    8    2	(JS) S&P 500 Volatility Oscillator For Options 2.0
     6    3	(JS) Tip-and-Dip Oscillator 2.0
     0    4	(JS) RSI Divergence Oscillator
#    8    5	(JS) Pivot Point RSI
#    8    6	(JS) Fibonacci Moving Averages Ribbon
     6    7	(JS) Bollinger Band Momentum Average Trailing Stops
     5    8	(JS) Bollinger Bands Momentum Average
     5    9	(JS) Sustained Velocity Squeeze Index
     6    10	(JS) Multi-Time Frame Pivot Point Detector

#   10   11	(JS) Squeeze Pro Arrows
    0    12	(JS) Squeeze Pro Early Signal Candles
    5    13	(JS) Squeeze Pro
#   7.5  14	(JS) TTM Reversion Bands
    0    15	(JS) TTM Trend Candles
    4    16	(JS) TTM Wave
    6    17	(JS) Candlestick RSI
#   8    18	(JS) Sustained Velocity Index
    0    19	(JS) TTM Squeeze Percentage
    7    20	(JS) Moving Average Secuirty Oscillator

    0    21	(JS) Moving Average Crossover Oscillator
    5    22	(JS) Tip-and-Dip Oscillator
    6    23	(JS) Trend Background
    4    24	(JS) Fibonacci RSI 1.0
    7    25	(JS) S&P 500 Volatility Oscillator For Options
#   8    26	(JS) Ultimate Multi Time Frame Moving Averages
#   8    27	(JS) Relative Volume Oscillator
    7    28	(JS) RSI Divergence Volume Weighted
    5    29	(JS) DMI Bars - Volume Spikes
    0    30	(JS) DMI Bars - Trend Labels

```

**Comment by adenysenko** [comment]
```
Context: so my boss Steevan has a back pain, so he is not in office by work from home.
So it's a bit blocking me to get resolve on goto Ukraine.

1) What are chances to him be back in office tomorrow?

2) What is best way ask him on "working remotely" [1-4]
1. send sms 
2. phone call
3. message on skype
4. email

3) So I prepared several options on how to ask for permission to work remotely.
Check the best pitch to use [1-4]:
1. "visit"
2. "medical"
3. "remotely ok"
4. "remotely"

----

4) Best technical setup to trade markets [1-10]
1. rsi divergence
2, cci divergence
3. obv divergence
4. stochastic divergence
5. money flow index divergence
6. awesome oscillator divergence
7. momentum oscillator divergence
8. macd divergence
9. Knoxville 
10. ppo divergence
```

**Comment by adenysenko** [comment]
```
4) Best technical setup to trade markets [1-10]
#	7.5	1. rsi divergence
	0	2. cci divergence
#	9	3. obv divergence
	6	4. stochastic divergence
	5	5. money flow index divergence
#	8	6. awesome oscillator divergence
	0	7. momentum oscillator divergence
	4	8. macd divergence
#	8	9. Knoxville 
	7	10. ppo divergence

```

**Comment by adenysenko** [comment]
- todo2_strategies.md
```
###########################################################################

**PPO_LH** (next candle)
====================================================
- setup
	- https://www.tradingview.com/script/CFMteznK-PPO-Divergence-Alerts-2-0/
		- "show PPO high-low triangels"	
- where
	- USDCAD 1D
- what
	- next candle should be same color as signal
- filter
	- especially good after a strong previouse move ###
		- many candles
		- or distance
- extras
	- PPO near DARWAS top - after rejection
		- if thin-box - go trend wise
	- confluence with LOH-HOL
	- PPO confluence on BRIGHT SQUIIZZEE candle - storong reversal!!
		- https://www.tradingview.com/x/6Qz1zwQO/
	


**HK-+-PPO_LH+50MA** (CONTINUATION - VERY GOOD) ####
====================================================	
- setup
	- HK
	- https://www.tradingview.com/script/CFMteznK-PPO-Divergence-Alerts-2-0/
		- "show PPO high-low triangels"	
	- 50MA
- confluence
	- PPO low high direction
	- trend
- what to do
	- simple
		- next candle should be color of PP signal


**HK-+-PPO_LHTREND** (REVESAL-1CANDLE - VERY GOOD) ####
====================================================	
- setup
	- HK
	- https://www.tradingview.com/script/CFMteznK-PPO-Divergence-Alerts-2-0/
		- "show PPO high-low triangels"	
	- Rubber Band
	- squzziee
- confluence
	- PPO low high direction
	- touched top of RubebrBand
	- extra
		- atfter many trend HK candles of the same color 
		- no squaizze (low volatility)
		- REJECTED WIGS from RubberBands on this candles or before #####
- what to do
	- simple
		- next candle should be color of PP signal
		
	
```

**Comment by adenysenko** [comment]
```
Context: looks like i'm good with working remote.

Lets check flights if any problems or delay on flights:

1. LOT-Polish Airlines 10:20pm flight on 13 December to Kiev

2. LOT-Polish Airlines  2:30pm flight to Nyc from Kiev on Dec 29

3. "Ukraine International" flight form Kiev to Zaporizhie at 8:20 pm on Dec 14

```

**Comment by adenysenko** [comment]
```
1) Best Awaresome Oscillator indicator strategy to use to make money [1-4]:
1. zero line cross
2. tween picks
3. saucers
4. divergence

2) Check if it worth to visit "I-MED" medical central for consultation.

3) Check me a girl "Maria"

```

**Comment by adenysenko** [comment]
```

1) Best next step to for making money on markets [1-4]
1. study more trading algorithms
2. start physical trading
3. start implementing strategy robots
4. start study AI to develop robots

2) Best homeopathic medicine course to take [1-19]
1	Glyoxal Compositum
2	Vertigoheel
3	Cerebrum Compositum
4	Cor sius Compositum
5	Lymphomysot
6	Heepel
7	Zeel
8	Testis Compositum
9	HeperHeel
10	Coenzyme Compositum

11	Tereodea Compositum
12	Engystol 
13	Solidago Compositum
14	Discuss Compositum
15	Mucosa Compositum
16	Ubichinon Compositum
17	Traumeel
18	Momordica Compositum
19 	Echinacea Compositum

```

**Comment by adenysenko** [comment]
```
1) What is best to do with my wisdom teeth on the upper jaws [1-2]:
1. repair
2. remove

2) Best homeopathic medicine course to take for my mama [1-6]:
1. Cerebrum Compositum
2. Vertigoheel
3. Tereodea Compositum
4. Solidago Compositum
5. Discuss Compositum
6. Echinacea Compositum

3) Check if it worth to subscribe to any tear in "Stock Market Mastermind" by ImpeccableInvestor [1-2]:
1. Student Plan
2. Master Plan

Context: I think to start to study chunks of new material now, so brain gets prepaired
4) Best thing to study for money making [1-2]
1. Studing Option trading
2. AI for trading


```

**Comment by adenysenko** [comment]
```
1) Best party to go for 31 December [1-7]
1 The Surrealist Ball: New Year's Eve at House of yes
2 Golden Era: New Year's Eve 2020 at 99 Scott
3 white lights - new year's eve by looktowardsthelight
4 sun-set by looktowardsthelight
5 The Cityfox Odyssey: NYE & NYD 2020
6 NYD 2020: Danny Tenaglia & Victor Calderone
7 New Year's Eve Time + Space 2020 ft Claptone & Shiba San

Context: My friend Mike is split with his gf Alena. It all good.
I want to defreaf situation with him "lessons learned", but need to make sure he does not tell me any BS.
2) What was the context/reason of the split? Can you check view on situation from both sides?

3) Check me girls [1-2]
1. polish girl
2. talina
```

**Comment by adenysenko** [comment]
```
1) Check what is the best for tonight [1-3]
1. wait for Anton to get some party access
2. buy tickets for **"Golden Era: New Year's Eve 2020**
3. preorder tickets for the "The Surrealist Ball"
```

**Comment by adenysenko** [comment]
```
Context: I can "NASDAQ 100 All Stars" trading package" from original developer Marketgauge or hacked cheaper copy. 
Not sure if it makes difference in quality or outcome.

1) Where is best to buy [1-2]
1. from Marketgauge
2. from "seller-store.com"

2) Best "Overnight Profit Strategy" trading strategy package to buy from "seller-store.com"
1. Basic
2. Pro

Context: this suppose to be trading mentoring groups
1) Best SimplerTrading trading membership to jump in now [1-3]:
1. options
2. futures 
3. fibonacci queen



```

**Comment by adenysenko** [comment]
```
1) Best futures trading broker to use [1-5]:
1. ampfutures
2. OptimusFutures
3. Infinox Futures
4. fxflat
5. phillipfutures

2) Best country to open offshore corp [1-2]:
1. Hong Kong 
2. Singapore

3) Check if it worth going Miami for January20 holiday.

```

**Comment by adenysenko** [comment]
```
For my trip in Miami 15-20.

1) Should I:
1. go alone
2. with anton

2) What place to get [1-3]
1. hostel
2. apartment
3. room

```

**Comment by adenysenko** [comment]
```
Context: Anton has delays with investment from Russians,
on other side it January was good timing to expand apartments.

1) How long it takes to Russians to send first money?

2) If opening second apartment, how long it takes the business to bring good $$?

3) What is preferable actions to take from my side [1-4]
1. Enter with Anton as partner in a new apartment
2. Enter with Anton as partner in a new apartment till he gets stable and after exit
3. Borrow him $ for the new apart investment till Russian $ are coming
4. Do nothing, let Anton figure out himself


```

**Comment by adenysenko** [comment]
```
1) Best URL structure to use for "program" APi service [1-2]:
1. structured
2. flat
```

**Comment by adenysenko** [comment]
```
1) Best course on trading company earnings events [1-3]:
1. Ami Bbusaad Earnings engine
2. SimplerTrading Earnings Secrets
3. ChrisD Quantitative Earnings System 

2) For "program service" best to [1-2]
1. incude service descriptors in repo
2. do not include 
```

**Comment by adenysenko** [comment]
```
1) Best trading courses from "t3live" to make money [1-10]:
1 Quant Edge Home Study Program
2 The Newsbeat Bandits Program
3 Mark Melnick’s Algo Course
4 Trading the Pristine Method
5 Redler Ultimate Access
6 Scott Redler’s Path to Profits
7 Newsbeat Master Class
8 Strategic Swing Trader Course
9 Actionable Options Program
10 Elite Mentorship Home Study

```

**Comment by adenysenko** [comment]
```
1) Best day to go out [1-2]
1. 10
2. 11


```

**Comment by adenysenko** [comment]
```
1) I go out tonight.
Lets check girls in the context of the night
```

**Comment by adenysenko** [comment]
```
Context: To effectively test strategies I need to write an automatic script.
Not necessary production level, but something working and practical.

1) Best environment to do it [1-6]:
1. Forex Strategy Builder Professional
2. Easy Strategy Builder
3. FOREX TESTER TRADING SIMULATOR
4. Molanis Strategy Builder
5. EA Wizard
6. quantconnect

```

**Comment by adenysenko** [comment]
1) Check me girl "Leksa"

2) Best days to go out [1-2]
1. 17
2. 18
@adenysenko adenysenko commented 22 hours ago
1) For going out on 17, check what is the best: [1-2]
1. meet Leksa
2. go with Anton and let Leksa join

**Comment by adenysenko** [comment]
```
1). Best strategy for making money on trading stocks for long term swings [1-5]
1. 200 Bolinger model by unpicable investor
2. nasdaq100 all stars
3. "quality company in uptrend" by quantopian
4. stem.mr model from sentiment trader
5. longgame by simpler trading
```

**Comment by adenysenko** [comment]
```
1) check me girls
1. anna
2. oksana
```

**Comment by adenysenko** [comment]
```
1) Best trading strategy to make money [1-2]:
1. t3 by tradingfibz
2. s3 by tradingfibz
```

**Comment by adenysenko** [comment]
```
1) Best day to meet Leksa [1-4]
1. 22
2. 23
3. 24
4. 25
```

**Comment by adenysenko** [comment]
```
2) In rightlinetrading trading system they use several indicators, let's peek which are most important [1-8]
1. RL - Dots
2. RL - Quant Lines
3. RL - Three Line Indicator
4. RL – Bias
5. RL – Exponential Oscillator
6. RL – Continuation Entry
7. RL – Revolutions Entry
8. Unirenko Bars


```

**Comment by adenysenko** [comment]
```
	1) Best indicator to use in trading [1-4]:

	1. Murray Math Levels by Shizaru
	https://www.tradingview.com/script/xdTyx8vM-Murray-Math-Levels/

	2. Murrey Math Lines by ceyhun
	https://www.tradingview.com/script/So9Q82mE/


	3. Murrey Lines v1.0.0 by ckryza
	https://www.tradingview.com/script/x8ZRlmof-Murrey-Lines-v1-0-0/

	4. [RS]Murrey's Math Lines Channel by RicardoSanto
	https://www.tradingview.com/script/PhluTmKF-RS-Murrey-s-Math-Lines-Channel/
```

**Comment by adenysenko** [comment]
```
1) Check me girls [1-2]
1. Suzanne 
2. Rose

2) Best day to meet Suzan [1-2]
1. tonight 
2. tommorow 

```

**Comment by adenysenko** [comment]
```
1) Suzan aside, is it worth going out tonight?
(basically i can just go alone or with anton out after meeting her)
````

**Comment by adenysenko** [comment]
```
Context: already interacted with this two girls, 
lets see who will be easier to get to sex

1) Girls for me [1-2]
1. Leksa
2. Karolina
```

**Comment by adenysenko** [comment]
```
Context: for Mastercard

1) Best way to inject Program parameters to business services [1-3]
1. Value Injection
2. Generic Capability Injection
3. Mapped Capability Injection

```

**Comment by adenysenko** [comment]
```
1) Best day to meet with Leksa [1-4]:
1. 5
2. 6
3. 7
4. 8

```

**Comment by adenysenko** [comment]
```
1) Best services/subscriptions for money making [1-2]:
1. The Kelly Letter
2. Adam Khoo Ultimate Investors Playbook


2) Best day to go out [1-3]
1. 6
2. 7
3. 8


```

**Comment by adenysenko** [comment]
```
Context: so girls Leksa is divorced but has small baby which add complexity.
We met couple of times for cooffe/food.

1) Check if it worth to visit Leksa at her home today?
(not sure as she has care of baby and hairy pippo can leave child physiologically disturbed )

2) What is the generally best approach to fuck her [1-2]
1. get her to my place
2. at hers

```

**Comment by adenysenko** [comment]
```
1) best day to meet Leksa [1-2]
1. 13
2. 14
```

```
1) best trading education membership to join [ 1-2]
1. sampler trading options
2. sampler trading futures
3. my investing club
```

**Comment by adenysenko** [comment]
```
1) Best day to go out [1-2]:
1. 14
2. 15

```

**Comment by adenysenko** [comment]
```
1) Generic reading for March

2) Card and mantra for March
```

**Comment by adenysenko** [comment]
```
Context: so Leksa is inviting me to visit her in the evening.

1) Best day to visit her [1-2]
1. 16
2. 17

```

```
1) Check me girls [1-2]:
1. Angela
2. Kaylee

```

**Comment by adenysenko** [comment]
```
1) Best day to go out [1-2]
1. 21
2. 22
```

**Comment by adenysenko** [comment]
```
1) Best day for a trip [1-3]
1. 23 feb
2. 29 feb
1. 1 march
```

**Comment by adenysenko** [comment]
```

Context: I recommend my Ukrainian tarot guy Igor to Alexa to work with. I gifted her couple of first session as she is a bit in trouble.
I one side I want to help her, on other side I do not want to get in some "ongoing dependency"
4) What is the best to do:
1. Just cover he sessions with Igor for some time
2  Just cover he sessions with Igor for several sessions
3. Let her to get fee agreement with Igor and after I cover her for some period of time
4. Let her to get fee agreement with Igor and I cover her initial welcome amount
5. Let her to get fee agreement with Igor and I cover her for several sessions


2) check if my 2018 Profit/Loss accounting statement for Alpha Proxima is ok to submit? If any issues?

Context: For my HK company I have a difficulty to find name alternatives. Just a bit no phantasy, but works need to be done.
3) Let's check best to do for company naming [1-2]
1. Go with "Xyno" name
2. Look for another name

Context: so I be opening HK company  via "Neat Bank".
As part of applying process you can specify a website for the company
4) Is it worth to create a site for the new company or makes no difference for them? Let check how many points diff


```

**Comment by adenysenko** [comment]
```
Context: so I work with Indian team and during debate I mentioned if you can do it - "just say NO if you can't do it, as it's possible that in india you culturally tend to say yes just to please".
This got peeked up as something they do not like.

1) Lets check [1-3]
1. did it was actually unsensible from my side
2.  its some politics from their side
3. mix 

```

**Comment by adenysenko** [comment]
```
1) Best night to go out with Anton [1-2]
1. 28
2. 1

Context: for HK company name let test adding "Limited"
2) best to do for company naming [1-2]
1. Xyno Limited
2. Look for another name

```

**Comment by adenysenko** [comment]
```
1) The best agent to work with [1-4]
1. Alex
2. Irma
3. Lev
4. TD travel insurance
```

**Comment by adenysenko** [comment]
```
1) The best agent to work with [1-4]
1. Alex
2. Irma
3. Lev
4. TD travel insurance
```

**Comment by adenysenko** [comment]
```
Context: "Realvision" is a service with financial market  analysis.
1) Best subscription to buy from Realvision to make money [1-2] 
1. essential
2. plus

```

**Comment by adenysenko** [comment]
```
1) Best  indicator to make money [1-29]:
1	Absolute Strength Histogram
2	Waddah Attar Explosion LazyBear
3	Waddah Attar Explosion V2 SHK
4	RS Fractals V8
5	Sweetspot Gold RN by JustUncleL
6	Sweetspot Gold R4 22 by JustUncleL
7	GM PRO TDFI
8	GM Lite Baseline
9	GM PRO Baseline
10	GM PRO EVZ

11	GM LITE Rex Oscillator
12	GM PRO Better Volume
13	GM Lite SSL channel
14	GM PRO Money Management
15	Trix Ribbon BB Bands GM
16	Ravi GM version
17	GM Improved RSI
18	GM Momentum Oscillator
19	Buy Sell Indicator by Goran Mateski
20	CCPR by Raghu

21	CCPR Next Day by Raghu
22	Raghu 9 30 Candle Target Hit Checking Indicator
23	Raghu 9 Indicators V1
24	9 15 Candle High Low Indian Standard Time
25	Raghu 7 Indicators V1
26	Raghu Intraday Excellent Indicator Final 2020
27	Raghu Camarilla with Buy Sell Zone Testing Period
28	BO Swing Finder R0 6 by JustUncleL
29	Volume Zone Oscillator

2) Best trading system to build on top [1-5]
1	Scalping Swing Trading Tool R1 4 by JustUncleL 
2	Price Action Trading System v0 3 by JustUncleL 
3	Pullback Trading Tool R5 65 by JustUncleL
4	Backtest QQE Cross v6 0 by JustUncleL
5	Pullback Trading Tool ALT R1 0 by JustUncleL

3) Best futures trading setup [1-8]
1	80% rule for value area
2	Fubar Sneaky
3	Initial Balance Breach 
4	Pit bull low trade setup
5	12:30 trade setup
6	1:00 trade setup
7	Open Tick gap fade
8	Ragee Trend Continuation setup


```

**Comment by adenysenko** [comment]
```
1) I have some sneezing and coughing ;) 
Let check for corona virus. 
```

**Comment by adenysenko** [comment]
```
1) Best course to take from simlerTrading
1. Submarket Sonar
2. Elliott Wave Masters Series
3. The VWAP Max Tool Package

```

**Comment by adenysenko** [comment]
```
1) Best trading system to use as base [1-3]
1. vzo trading system by dave stakler
2. vzo and pzo devergence
3. vzo and price devergence
4. obv devergence 
```

**Comment by adenysenko** [comment]
```
Context: I will be working remote next period of time, so I think if there is better way to do it.

1) Best thing to do for me the next 2 weeks:
1. stay in current Jersey apartment 
2. stay in Florida 

```

**Comment by adenysenko** [comment]
```
Context: so still in planing should we go Florida or not

1) How about rent some house in small city not necessarily Florida

Context: the current appt building we have has several place with office setups. They are still somewhat shared.
2) Which of them is safer to take [1-2]
1. 3 floor
2. 2 floor

3) Best dumbells to buy [1-3]
1. Bowflex SelectTech Adjustable Weights
2. ATIVAFIT Adjustable Dumbbell 

```

**Comment by adenysenko** [comment]
```
1) how long it will take Neat to create HK company if submittig now? weeks
```

**Comment by adenysenko** [comment]
```
1) lets check again [1-2]
1. stay home in Jersey City with Mike
2. go florida with anton
```

**Comment by adenysenko** [comment]
```
1) Best thing to me to focus today for my Mastercard work today [1-3]
1. Work on Api Standards page
2. Finish  Api Standards page
3. Start working on Logging and Monitoring page

```

**Comment by adenysenko** [comment]
```
Context: working on "logging" analyses for Masrtercard

1) Is it good to send the current version?

```

**Comment by adenysenko** [comment]
```
conext: so some opportunities with Hashmel with protective masks.
Lets see if it worth do any of that:

Best option with masks speculating [1-3]:
get a supply of masks delivered for my company name
give hashemel $ so he manages it
not get involved
```

**Comment by adenysenko** [comment]
```

Ziggata: placement in mastercard
    - 10 chelovek - 300 
    - 


Nado sostavit profil na kompaniju, 
dlia silnoj presentazi i podojti k etomu metodichno.

Esli u cheloveka est poniatie togo chto vy mozhete emu legcge najti rabotu libo sozdat rabotu dlia vas.

EXECUTIVE SUMMARY

TIP1: most in demand now + high margin
    - front-end: angular, vue
    - beckend: node, java(spring, spring-cloud) 
    - cloud: amazon, ms, google

TIP2: 
    - financial and large comoanies - java backend, js frontend ##

TIP3:
    - osnovnoj upor budet na "working technologies" - tam gde delajutsia $$$$
    - ai 

TIP4:
    - amplifcication
    - quality and reliability



- scentific expertise:
    - 

- technological epertise po firme v zelom 
    - backend: node (....), java (spring,spring-cloud, spring-mvc, spring-rest )
    - front-end: angular, vue
    - cloud
    - ai: tensorflow (,,,)

- spisok prokectov
    - project
        - period
        - bissness description
        - what was done buy the company
        - technologii envoled:
        - infrastructure involved:  orcle, sales force, .....
        - administrative involved: jira
    -

Nahid Giga

5757 Main Street, Suite 200,
Frisco, Texas 75034

#mantra 
    - opportunities are unlimited
    - nado partners
    - BIZDEV - zainteresovany bolshe v videniee business chem vedenii technologiie
    - QUALITY PARTNERS

- seichas kak by on hold busdev - poeotomu smotrit najti partner remotely  
    - luchshe: najti solid partners potom hujarit projects

# industries: financial, banking, automaive, helthcare, commerce, data meneagment, marketing cloud, commerce (supply manangemnt)
    - BLOCKCHAIN - not that much - protos

# technology: backend(java, node), frontend(angular, vui), ai (prototypes)

#area
1) project delivery
    - solid team
    - competitive rates
    - latin america, eastern europe
2) stuff augmentation
    - stuff comes here and worke
    - remote work
    - #proficency in english

Roman is belorus 




PACIFIC UNIONFinance 
GM FINANCIALFinance 
CALIBERFinance 
GOODMAN NETWORKSTelecom 
DAISY BRANDManufacturing 
ENCORE ENTERPRISESReal Estate 
HILLTOP HOLDINGSFinance 
TURBO CHEFManufacturing 
ARGO DATASoftware H
OWARD HUGES CORPORATIONReal Estate 
GAMESTOPRetail MR. COOPERFinance
PLAINS CAPITAL BANKFinance 
AT HOMERetail TOYOTAFinance 
LENNOXManufacturing 
MOTORSPORT AFTERMARKET GROUPEcommerce 
BLUESTAR SPORTSSoftware 
PRODEASoftware MATCH.COMWeb Platform ENERGY TRANSFEROil & Gas 
DIGITAL REALITYReal Estate 
XOMEWeb Platform 
SECURUSSoftware 
DHISCOSoftware GENERAL 
DATATECHTelecom 
SANTANDERFinance 
PROVISTAHealthcare ARMORSecurity ELEAD ONESoftware 
FUJITSUSoftware GAINSCOInsurance VIZIENTHealthcare SHORTS TRAVELServices SOLERA HOLDINGSTechnology Vistra EnergyEnergy TXU ENERGYEnergy USA FOOTBALLSports 



```

**Comment by adenysenko** [comment]
```
Context: buying some 
hard drive backup system 
on Craigslist from 
the guy with 
6462099173 number.

0) check if the guy is reliable

1) check if his 20TB Ride is good to buy

2) check if his 12TB Ride is good to buy

```

**Comment by adenysenko** [comment]
```
JUNE 3

Morning Daytrading Recap for RTY YM NQ (video) https://youtu.be/q6XxqXs0A-k

Thursday Session Notes https://workflowy.com/s/simpler-futures-thur/fP8G15omh0YfA8Jq 

The NQ did have a 15-minute Breach and while that is valid, the clearing range breach never came so immediately this is laggard to the rest of the market. 
So what is another way I could handle this situation specific to the NQ. 
## If the clearing range breach does not happen by 10:30 AM ET, cancel the the 15-minute open orders. #####

-----
https://www.tradingview.com/x/zHqvW4lC/

aggro:  last DARVAS candle close ???
conservative: 930Vwap

RH - 06/04/2020 @ 10:12AM:
AGGRO BUY JUNE YM @ 26236 LIMIT [TO OPEN] MODERATE BUY JUNE YM @ 26207 LIMIT [TO OPEN]  CONSERVATIVE BUY JUNE YM @ 26135 [TO OPEN] STOP LOSS SELL JUNE YM @ 26115 STOP [TO CLOSE] TARGET 1 SELL @ 26260 LIMIT [TO CLOSE/SCALE OUT]  Note: I'd rather have a stop loss below 26100

---

https://www.tradingview.com/x/QjcdVNCK/

NQ JUNE 3

Raghee : BULISH Bias
Mine: short bullish (medium yen, good bonds/vix)


15M breach
- SOSO
- same direction as macro bias
- we have 10+5
- enter on bullish PIN bar


30M breach never happened:
- BUT RISKY entry
- 930 Vwap
- lg_home
- enter on bullish MS
- hold on: risk_on, risk_on_rsi

------

ES JUNE 3

https://www.tradingview.com/x/2py7mN23/

30M Breach is confirmed
we want to enter from PB from historical 1 level on 930 vwap
exit on DJ at vwap 930 2std level,
or we can hold as risk_on 
and risk_on_mementum are still growing 
also we seen higher Vwap levels to bouce



```

**Comment by adenysenko** [comment]
```
lucidfx devergence
- https://www.tradingview.com/x/P0yJ2xqD/
- https://www.tradingview.com/x/MALBaPzn/

bolshe - kakaja verojatnost togo chto devergency ostanetsia? ###
nesrabotajushja divergence before : https://www.tradingview.com/x/os09ICtl/

```

**Comment by adenysenko** [comment]
- https://www.financialjuice.com/home
- https://intercom.help/simpler-trading/en/articles/3769494-watchlist-raghee-watchlist-column-code-and-shared-links

**Comment by adenysenko** [comment]
## VSCORE BREACH AND RETREAT
```

- wait for PriceAction to confirm the bounce started: "T" or other candles - not sure 
    - https://www.tradingview.com/x/cFoqhm6n/  - "T"
    - https://www.tradingview.com/x/RWeLlunO/ - "SB"
- wait for red LG, tho it maybe a late wait 
    - https://www.tradingview.com/x/RWeLlunO/

- should you close on the first extreme riches after bounce?
   - does it do gradual bounce?
   - or single bounce to top and fuck it?

 -------

JUNE 15 - risk on

min(vscore, 9:40 - till 30 mins is breached ) trailing
max(vscore, 9:40 - till 30 min is breached ) trailing

if bias is positive:
we buy then
    vscore < vScoreMin
we sell then
    vscore > vScoreMax

-- we enter after 1000, only if we have 30 mins breach
--we may sit on the position - as we can have some stagnation ####
-- if we do not breach 60M till 1130 then we cancel the bias

- ES1! https://www.tradingview.com/x/z28f7j7v/
- RTY1! https://www.tradingview.com/x/J5hmzM4r/
    - later https://www.tradingview.com/x/xc3ct3OG/
- YM1! https://www.tradingview.com/x/0VlObWv4/

Hypotehesees:
- if Risk On, then we buy from 9:30 Vwap?

=====

JUNE 15 - risk off
- 15 mins breach UP, 30min flat, 60 mins breach down!

- YM1! https://www.tradingview.com/x/bcxxuKMh/
 -- sell off, no retest
- RTY https://www.tradingview.com/x/EcmfLeIv/
 -- sell off, no retest


=====
JUNE 10 - risk off - mixed

- RTY1! https://www.tradingview.com/x/8NnOPRzI/
--- found sell level but it was not holding well, market was bulish but sold off only after 2pm

- ES https://www.tradingview.com/x/cFoqhm6n/
--- found sell level but it was not holding very well
--- check vscore bounce confirmed by PA candles
- YM! https://www.tradingview.com/x/RWeLlunO/
-- same situation


```

## PROPS
```
===== PROPS

JUNE 10 - bearish bias
- RTY! - https://www.tradingview.com/x/8NnOPRzI/
-- 15 min breach on strong trend and retrace into 34 and bounce sown, but bias is BerishMixed


==== iDEA
- afternoon trend with IBB bias and MACRO confluence!

```


## VSCORE DAY RESTEST =======

```
bounce from zscore with day bias ##

RTY - https://www.tradingview.com/x/ZOq5Idm0/
    13:55
    15:35
ES - https://www.tradingview.com/x/LyBp4Z5d/
    - 15:35
-YM - https://www.tradingview.com/x/a0Er011H/
    - 15:35

    
```

**Comment by adenysenko** [comment]
## VSCORE DAY RETEST =======

```
bounce from zscore with day bias ##

JUL 16 
- RTY - https://www.tradingview.com/x/ZOq5Idm0/
    13:55
    15:35
- ES - https://www.tradingview.com/x/LyBp4Z5d/
    - 15:35
-YM - https://www.tradingview.com/x/a0Er011H/
    - 15:35


JUL 16 -
    - sentiment afternoon/moorning is berish
        - plus Pauls is speaking
        - https://www.tradingview.com/x/5DrmuOr3/
    - NQ1! IBB bearish 
        - retest extreem VScores top at     - 20:00 and 21:00
        - https://www.tradingview.com/x/KWVCJ6fv/
    - all are bearish, but only NQ support extreems
        - https://www.tradingview.com/x/gqYAhE1a/

```

**Comment by adenysenko** [comment]
```
1) There is a meeting today with Diiptech guys on role distribution. See should I
1. participate today
2. pospone to Friday


2) best trade sistems from ASFX [1-3]
1. D1
2. D2
3. A1

```

**Comment by adenysenko** [comment]
Know your futures basic - Part 1
https://workflowy.com/#/79c9129c5d43


- https://s3.amazonaws.com/cloud-streaming/chatrecordings%2FSFGold/1008_AlertLog_02-17-2020__09.19.539_AM.txt
- https://s3.amazonaws.com/cloud-streaming/chatrecordings%2FSFGold/1008_AlertLog_01-06-2019__09.43.142_PM.txt
-

**Comment by adenysenko** [comment]
```
1) Bet subscriptions/courses for money making on markets [1-4]
1. Macro Ops Collective Subscription for 1 Quarter
2. Adam Khoo Investor Playbook for 1 year
3. Macro Ops  ValueVentures
4. SimplerTrading Options membership

```

**Comment by adenysenko** [comment]
- https://www.koshyjohn.com/software/memclean/

**Comment by adenysenko** [comment]
- 23 JUNE 2020
- sentiment 
  -- mine - mixed - https://www.tradingview.com/x/6mrOoIrs/
  -- finviz - mixed - both Jen and AUD/NZD  are up, but AUD/NZD
  -- news - not bad, nothing n
- markets
   -- ALL mixed - https://www.tradingview.com/x/PSaaCcev/
-- Raghee - we are in asset buble, so we going to look long, focus on NQ - which is strongest, or we buy from ES down extreems
   -- ES we could buy because of NQ is driving market up, and ES has similar structure
   --  RTY is generally not so good with recovery, even if it riches Vscore exreems faster
--

**Comment by adenysenko** [comment]
# 2 chasa - dohuja vremeni - nado imet paru STRATEGY - not just retreat

**Comment by adenysenko** [comment]
- for PM reversal make sure LG is lightly reversed
- green pointing up
- red is on other side
- blue maybe
- https://www.tradingview.com/x/XwPsTcxQ/

- do not enter on blind VScore bound especially if other INDEXEs are down

**Comment by adenysenko** [comment]
# REVERSE from VSCORE extreems
-- https://www.tradingview.com/x/OvRoG0oO/
-- make sure GREEN left bottom 
-- GREEN crossed bottom UP on strong volume

--

**Comment by adenysenko** [comment]
-- june 24

# ideas
- **simultanginoal breach**
-- if all indexes breaching down on 9:30:10:00 near same time then we can short till 10:05
-- https://www.tradingview.com/x/1djxLMu0/

**Comment by adenysenko** [comment]
#news sites
- https://www.fxstreet.com/news?dFR[Category][0]=News&dFR[Tags][0]=Equities

**Comment by adenysenko** [comment]
- JUNE 25 

**ROOM NOTES**
``
- NQ are not breaking up on 30M
    - but XLF and XLV 
    - are broken to broken up
    - so we can trade ES
- on HPMRs lines
    - low - dotted line
    - average - 
    - high - 
- Use options - ATM or slightly ITM    
- FOR 3:30 alerts 
    - for late in the day trades 
    - use options on futures
- your safe place is EXTREEMs
    - whenever you have Berish or Bulish bias
    - the unsafe pace
- assume you gonna to loose then you do you position sizing
    - assume you gonna loose ###
- props and emas
    - more help in second half of the day
    - first half of the day more helpfull - hpmr, vscore 
- previsouse H/L, darvis level, psyho-levels
- do not looks for adrenalin rush
    - could be a day without a dyly setup ###
- Raghee
    - artur: in worst case look at VAH/VAL ??? ###
- REATREATS ARE good 
    - but we want to focus on VSCORE EXTREEMS and HPMRs ###


**ARTUR NOTES**
```
- ideas
    - LG - better GREEN and BLUE are in sync to enter on RETEST
    - buy from pin bars - on high valume ####

- conclusions
    - #### FIXIT
    - # 9:40-10:20 ranges are too extreem
        - 9:30 drop can be too strong and has no prcedents before in pre-market 
            - so you will not get low to this level later
            - NQ https://www.tradingview.com/x/CW3wjccJ/
    - # EXTREMES
        - buy/sell from extremes - VAL/VAH, hpmrs, vscore ##
    - BUYS
        - in "not so bullish" do not buy RTY
        - buy NQ and ES
    - 7:00-9:00 looks like more effective
        - but they are MORE agressive
    - look for conflunce
        - vscore levels
        - VAL/VAL
        - volume candle: PIN, ....
        - LG: greeen and blue in the same area
    - there is expiration of the IBB
        - no confluence by indexes
        - no new breches clear by 11:00
        - then exit existing
    - go conservative
        - RTY levels https://www.tradingview.com/x/uOUVgS47/
            - agro - failed
            - conservative -  worked ###
        - "not so sure day" => go conservative #####
```
hypotheses
```
    - IBB
        - # NOT SO SURE ON BULLISH-NESS DAY - BURST UP - but it was short leaved
        - sentiment
            - all https://www.tradingview.com/x/RwO6S0Si/
            - risk currency are slightly up
            - but bonds are higher too
            - some concerns on CORONA
            - end of month? rebalancing? ##
        - # non-confluence
            - 15 min breach DOWN - across all
            - 30 min breach UP - across all
            - 60 min NO breach - across all
        - thoughts
            - expiration of breach
                - after 10:00 breach
                - if all indexes are not breached 10:30
                - then close your postion by 10:30/11:00 opened with bias based on 10:00 bias
                - ALL https://www.tradingview.com/x/9SLHmVon/
                - RTY https://www.tradingview.com/x/uOUVgS47/
            - LG trend is valid
                - better GREEN and BLUE are in sync to enter on RETEST ?
                - (not sure if its too late)
            - entering from 7:00-9:00 levels
                - I enterd LONG from 7/9 high = very agressive
                    - it was in correspondance with 9:30 VWAP
                    - RTY https://www.tradingview.com/x/uOUVgS47/
                - better to enter LONG 7/9 LOW 
                    - if its a still valid trend
                    - it also can correspond from VAL ???
                    - ES https://www.tradingview.com/x/jRGK7sOo/
            - enter from VAL ??
                - ES https://www.tradingview.com/x/jRGK7sOo/
```

**Comment by adenysenko** [comment]
```

- macro sentiment
    - iznachalnaja reakzija
        - naskolko ona budet silnoj i dolgoj
        - #stoit li FOMO?
    - pullback
        - naskolo on budet silnym i dolgim
        - #stoit li byt "contrarian"

**CONCEPT**
    - breached 15M
        - we start to look for it to bounce from opiste side
    - breched 30M
        - preciouse cancelled
        - we start to look for it to bounce from opiste side
    - breched 50M
        - preciouse cancelled
        - we start to look for it to bounce from opiste side


**PRICIPLES**
- do not enter MOMEMNTUM at VSCORE extreeme
    - range [1, -1]
- if MOMENTUM entry
    - then NO PULLBACK???? ###
- NO BREACH - no entry ###
    - #IDEALLY waiting for 2 breach confirmation
    - if 15M not breached - we are carefull to enter EARLY as situation is unclear
        - waiting for 30M and then enter
        - M2KU2020 JUNE 22 - https://www.tradingview.com/x/F1DQ1OTn/
- the futher in time - the deeper VWAP retracemnt should be ##
- 9:30>7>2 VWAP "279" by 10am?
    - wait for "279" to allign
    - #must be 
        - pointing the same direction
        - expanding (9:30 is leading and on top/bottom depends on trend)
    - exampeles
        - how about this?
            - M2KU2020 22 JUNE https://www.tradingview.com/x/xUgjotRC/
                - pointing the same direction
                - no "279"
                - not expanding


**ENTRY_CONFUENCE**
    - elements       momentum
    - 279               x
    - 279<              x
    - 279samedir        x
    - 15IBB + 30IBB     x          
    - 930wvap           x
    - 38fib             ?
    - darwis_volume     x

**ENTRY**

# 15 MIN - COLOR
    1) retracement to 9:30Vcore and fib (immediate 30M breach after same-color 15M breach)
        - logic
            - trend most probably is strong
            - we have a retracemnt
                - but low probability it will REACH the other side of Vscore range
        - look for confluence:
            - time
                - 10:30?
            - risky
                - 9:30 vwap
                - atleast 38% retracemnt
            - risky confirmed
                - wait till darwas is broken down with VolumeSpike
                    - you will still have time if trend is strong
                    - MESU2020 - JUNE 24 - https://www.tradingview.com/x/snMUvwHM/ #279
        - examples
            - MESU2020 - JUNE 24 - https://www.tradingview.com/x/snMUvwHM/  #279
            - M2KU2020 - JUNE 12 - 10:21 - https://www.tradingview.com/x/ngYFOTSn/ ~279
                - darvis_volumne+vwap930
        - average
            - MESU2020 - JUN 18 - https://www.tradingview.com/x/H4SPyTjq/  @279
                - 930vwap+38 worked, but volume confirmation was a bit later
                - JUNE 18 was risk OFF day so up was chunkie
            - YM1! - JUN 10 - https://www.tradingview.com/x/uX5EF7pK/ @279
                - darvis_volumne after vwap930
                - a bit late entry
                - vscore=2 - no place to go, shallow

    2) 10:00am level brake (immediate 30M breach after same-color 15M breach)
        - logic
            - momentum is strong we may ride it for short time
            - TS = atr(m1, 4)
        - super Aggresive
            - broken
                - 10:00 level is broken down with Volume spike
                - darwas broken down with Volume spike
                    - darwas is better - more establieshed, good if corresponds with 10am level
            - maybe
                - GRAB candles
                - prop dots?
        - time
            - 10:00-10:10
        - examples
            - MNQU2020 - JUNE 26 - https://www.tradingview.com/x/qNUpnWLP/  #279
            - M2KU2020 - JUNE 26 - https://www.tradingview.com/x/4A4dxoNV/  #279


    3) PROP DOTS on 15M breach (????)
        - M2KU2020 - JUNE 26 - https://www.tradingview.com/x/4A4dxoNV/ #279
        - MESU2020 - JUNE 22  - https://www.tradingview.com/x/XO8QBYPX/ @279


# 15 MIN - YELLOW
  
   - 30 MIN breach
        - we enter moomentum on ly if 279 is aligned

    3) 30 MIN brech of 10am level
        - MESU2020 JUNE 26 https://www.tradingview.com/x/qpMwlLYp/
            - 279
            - 279 expan
            - 279 diretion
            - darwis volumen
            - 10am level
            - vscore = 1.6

    4) 30 MIN breach + momentum 
        - good
            - M2KU2020 - JUNE 12 https://www.tradingview.com/x/rQheFUna/
                - ~279
                - darwas_volume
                - 10:26
                - bounce from 930vwap
        - not clear
             - M2KU2020 JUNE 22 - https://www.tradingview.com/x/F1DQ1OTn/
                - going GREEN, but 
                    - RED > PINK (0.9 vs 0.3)
                    - so the top is shallow ###
                    - but we were comming from (-1.6)


- 15 YERLLOW, 30 color, 60 COLOR
    
    -  moentum entry RISKY
        - 30 minut breach
        - pullback to 2 am
        - 279 are mixed
        - darwis_volume
        - MYMU2020 JUNE 12 - https://www.tradingview.com/x/nqGZz8yv/


- coreect pullbacks from 7-9

    - MYMU2020 JUNE 12 - https://www.tradingview.com/x/nqGZz8yv/  
        - 12:24
        - 30 and 60 are aligned
        - lime pullback
        - darwis volume

- pullback from 940-10_20
    - MESU2020 JUNE 25 - https://www.tradingview.com/x/YjSN6NIf/
        - not so good
    - 


```

**Comment by adenysenko** [comment]
# june 30
- https://workflowy.com/#/ebc6dcefb755
```
- Note the VScore Extreme Highs and Lows between 7:00-9:30 AM ET and 9:30-10:00 AM ET (this one has been increasingly important!) 
- Five tactics to keep an eye on...1) VScore only 2) HPMR only 3) VScore+HPMR "stack" 4) Momo w/10x and 3x8 Prop/Conviction 5) Breach/Retreat/Retest
- 8:45 AM ET Currency Lineup https://www.screencast.com/t/4gqXTwkEeRxM
- Is gap psychology is play? ES 3155.50/2962.50, NQ 10211/9691, YM 26217/24717, RTY 1468.10/1370.10

```

**Comment by adenysenko** [comment]
# system
- momentum entry only on break of prev IB level into same direction
- do not enter >50 > 38 retracement on IB15

**Comment by adenysenko** [comment]
# 2 july

- holiday affected day
- also trump talks at 10:00
- NFP was today so its a conceiuse day

**Comment by adenysenko** [comment]
# system retracement

- after MOM breakout we almost always have a RETRACEMENT
    - if we enter on MOM - we have to take hit of waiting
    - it may also be a failed breckout ##

- # TEST KC as levels ####
- after volume spike breakout
    - # PULLBACK AFTER BRECKOUT
        - do not enter 
            - if entry candle close bellow SIGNAL candel close
            - or they are in same levels

    - **1x volume**
        - enter
            - enter right away
            - wait till pullback 
                - FIB ???
                - # EMA 13 pulled back and crossed up#
                - EMA 21 
                - # second dot on EMA6 - AGGRESIVE!
            - wait till pullback and get to the same level
                - LIMIT order
            - if you see 10+5 then you ALREADY waiting for pullback -> just buy it
                - https://www.tradingview.com/x/zu0cYEhA/
            - examples
                - https://www.tradingview.com/x/ajdLw2KJ/
    - **2x volume**
        - enter
            - righ away
            - # EMA 6 pullback and crossed up
            - EMA 8 ???
        - examples
            - https://www.tradingview.com/x/ajdLw2KJ/
    - **2*1x volumes in a row (or periodic confirming)**
        - enter
            - # EMA 6 pullback and crossed up
        - examples
            - https://www.tradingview.com/x/Vv3R9aZ2/
            - https://www.tradingview.com/x/mLpRfAqh/
    - 1x volume and oposite 1x
        - expect DEEP pullback
        - hypotheses
            - especialy with OB/OS CFM
            - wait till CFM goes green
        - examples
            - https://www.tradingview.com/x/L08AzrE6/

- SL
    - right on breakout
        - EMA 23
        - previouse fractal L/H
        - avg(EMA 23 close + EMA 23 (top))
    - on retracemnt
        - MIN
            - bottom of signal candle!!
        - OR
            - if EMA  6 => EMA 13
            - if EMA 13 => EMA 23

- TRAIL
    - # more aggresive for breakout, less for retracemnt 
    - EMA 23
    - VScore 
    - ATR stops (14, 3,   hk=0.1)
        - ATR stops (14, 4,   hk=0.1)
        - ATR stops (14, 2.5, hk=0)
    - ATR traling stop (5,3)

**Comment by adenysenko** [comment]
# TODO
- momentum trades outside of IBB
- pullback to VWAP with IBB 60M bias 
-- "nado tolko verit?"
-- how can we sure it works?
-- what are requirements?

**Comment by adenysenko** [comment]
- https://www.simplertrading.com/daily/futures/daily-time-frame-trends-and-breakouts/
- june 15 on Cl we have 10+5 momentum so we can buy on pullback to 21ema and we had "Double green" too
- artur: **looks like our dmi is noot good** we have yellow instead of green ADX
- for breakoutvtrace on Cl we do not have volume "conviction dots" and breakout(darwus zone for prise resistance zone, 1months vprofile for market resistance zone)
- NQ fave use 2 pullback buys in june, but is it momentum env? no

**Comment by adenysenko** [comment]
# jul 7
- "picture perfect day" on ES and NQ, 
- **raghee notes - https://workflowy.com/s/tuesday-july-7-2020/zJAuqGJ2E4PkkC4H**
-- **Maybe the best notes ever :)**

- peeresmotret kak ona naznachaet targets. 
 --- skoree vsego po VProfile?
--- overnight/regular LH?
```
ES Levels to Watch: 
- Overnight High 3184, Overnight Low 3138.xx. 
- Regular Trading Hour High 3174.xx, Regular Trading Hour Low 3155.
If ES retests the overnight low with NQ not in meltdown mode, I would be open to a buy as I would expect a bounce. 
If the open rushes with volume at 3135.xx that could be a sign that the sellers are at the wheel. 
```
- alerts
```
RH07/07/2020 @ 10:17 am 
Sep ES Daytrade Buy. 
    AGGRO BUY @ 3162.25 LIMIT TO OPEN. (I would prefer ES or SPY calls from this level.) 
    MODERATE BUY @ 3158 LIMIT TO OPEN. 
    CONSERVATIVE BUY 3153 LIMIT TO OPEN.
    CLEARING RANGE LOW BUY 3148.25. 
STOP LOSS SELL @ 3147 STOP TO CLOSE.

RH07/07/2020 @ 10:21 am 
Sep NQ Daytrade Buy. 
    Red Dot BUY @ 10650 LIMIT TO OPEN (Preferably a NQ or QQQ ATM options) 
    AGGRO BUY @ 10633.50 LIMIT TO OPEN.
    MODERATE BUY @ 10610 LIMIT TO OPEN.
    CONSERVATIVE BUY @ 10585 LIMIT TO OPEN.
    CLEARING RANGE LOW BUY @ 10558 LIMIT TO OPEN. 
STOP LOSS SELL @ 10540 STOP TO CLOSE.

RH07/07/2020 @ 10:24 am 
Sep YM Daytrade Buy. 
    AGGRO BUY @ 26005 LIMIT TO OPEN.
    MODERATE BUY @ 25965 LIMIT TO OPEN.
    CONSERVATIVE BUY 25935 LIMIT TO OPEN.
    CLEARING RANGE LOW BUY @ 25925 LIMIT TO OPEN. 
STOP LOSS SELL @ 25890 STOP TO CLOSE.

RH07/07/2020 @ 10:55 am 
Sep YM Daytrade Update. 
    TARGET 1  SELL @ 25995 LIMIT TO CLOSE/SCALE OUT

RH07/07/2020 @ 10:57 am 
Sep NQ Daytrade Update. 
    TARGET 1 SELL @ 10670 LIMIT TO CLOSE/SCALE OUT

RH07/07/2020 @ 10:57 am 
Sep ES Daytrade Update. 
    TARGET 1 @  3167.50 LIMIT TO CLOSE/SCALE OUT

```

**Comment by adenysenko** [comment]
## JULE 8
- skoree vsego esli RUT bylo zelenym ranshe chem vse - znachit RISK ON
--- https://www.tradingview.com/x/Pp9fLG9A/
- na ES - bolee deshevye pozizi - poschitat s ATR i s point price?

**Comment by adenysenko** [comment]
# jul 9
-

**Comment by adenysenko** [comment]
```

**TODO**
===============================================================================================================================
- diiptech
    - legal language
        - exit cases
        - referal agreement
        - non-execution of commitment
            - vesting
    - referalls pochemu berem %
        - work
        - priveli klienta - potom ushli
        - skolko tam etih klientov budet?
        - #esli budem nadavlivat Roman budet obizahtsia, sho mol ne doplachivajut
- affise
    - #managit kto chto delaet i kak delaet ###### pohozhe Roma ne ochen hochet managit
        - leel: stoit li mne brat "middle v svoi hands?"
    - rabota s nimi
    - kakim makarom delaem firmu?
    - kakim obrazom delimsia?
- cybersoil
    - cto s nimi?

```

**Comment by adenysenko** [comment]
# 13 july hpmr buy

**Comment by adenysenko** [comment]
13 JULY 

- https://www.tradingview.com/x/3vNuJmNx/

- vscore bounce
--- precondition:  ADX = red, CMF = red, bounced from Fib 380
--- waiting for CMF going green (crossing up), or volume up spike
--- if ADX remains red or yellow  while CMF is green - be ready to exit = move is not strong
--- ADX must move into Green within 10-15 candles, 
---- best if it goes from RED to Green, without yellow
---- means strong reversal
---- if ADX goes green, but after yellow - exit, weak trend
--- keep till: (CMF remains green or Grab == green) and (ADX != red and Grab!=RED)
----

**Comment by adenysenko** [comment]
#14 

- **VZO or CMF?**
- https://intercom.help/simpler-trading/en/articles/3676832-about-turbo-vzo-signals

**Comment by adenysenko** [comment]
```

Context: so I need to start talking with Viktor
to sign an refferal agreement with his Sombra company.
I will be finding them clients.

Give me a general advice on conversation with him?

Should I wait for him to re-start conversation?

Do I need to collect more info before the talks?

Should i talk this week on next?

When is the best day to talk: 15, 16, 17 


Check when is the best to tal


```

**Comment by adenysenko** [comment]
```
Context: we are collecting legal information during user giving privacy consent to Mastercard.

1) Check what peaces of consent information 
is it  worth to store 
together with consumer record [1-3]:
1. consent type
2. service code
3. service function code


```

**Comment by adenysenko** [comment]
```
1) General reading for this week.

2) Card for this week.

```

**Comment by adenysenko** [comment]
```

Context: some Mastercard tech

1) For "consent type" enumeration what naming convention is best to use [1-2]:
1. uppercase full words
2. first letters abbreviation


2) In ConsentService methods whet is the best way to represent "consent category" [1-2]:
1. as string
2. as enumeration


```

**Comment by adenysenko** [comment]
#28 JULY
- **range breackout SETUP**
  -- https://www.tradingview.com/x/RCZNsB9O/
  -- DMI goes from yellow to PINK
  -- CMF was already RED color
  -- DARWAS brekout on volume
  -- SQ breakout

**Comment by adenysenko** [comment]
Context: my old friend Victoria is adding me back on Facebook

1) What does she wants?

**Comment by adenysenko** [comment]
# MacroOps
```
TODO FILTER: 
- less trades but higher impact or quality
- underlying: above/bellow WVAP(D/M/W), WVAP(D/M/W) rising/falling, ADX color
- internals: not in SQ

open: HYG/SHY 
close: VIX3M/VIX

buy: XLC(2.6), XLK(2.4), QQQ(2.87), FNGU(2.4), GVIP(2.6), RUT(1.9), SPY (2.03), JNK(1.8)
sell: UNG(12.6), QG1!(2.8), USO(2.04), CL1!(1.7), COW(2.7), PEX(2.1), PSP(2.8), XLE (1.7), XAE1!(1.9), ZC1!(1.6), JETS(3.4), JNK(2.7), AUDJPY(1.5), NZDJPY(1.49)

```

**Comment by adenysenko** [comment]
Contex: me and 2xRomans will organize work with company Affisse.
We will provide Affisse with lead generation and marketing support.

1) Best way to organize the work with Affise [1-2]
1. form a company for me + 2xRomans  
       and sign contracts between this company and Affise 
2. each of us signs separate contacts with Affise

**Comment by adenysenko** [comment]
- https://oec.world/en/profile/bilateral-country/usa/partner/chn

**Comment by adenysenko** [comment]
```
NCRSI (3 bars)
--------------------
    QQQ
        RAW buy = 1.63
        JNK/TLT buy = 2.1

    VXD
        RAW buy/sell = 2.8      ####
        RAW buy = 7.1           ####
        VXT/VIX buy/sell = 1.9

    VIX
        - RAW buy = 6.4

    CL1
        RAW buy/sell = 1.4
        VXT/VIX sell = 1.5

CRSI (3 bars)
--------------------
    - ORIGINAL
        - QQQ = 1.8
        - VXD = 2.26
        - SMH = 2.26
        - IGV = 1.5
        - XLE = 2.0

    - MODDY
        - XLE = 2.1
        - VXD = 2.26
    

```

**Comment by adenysenko** [comment]
```
Context: Mastercard tech stuff

1) Best design to load consent from AEM system  [1-4]:
1. independent 
2. independent with excessivity check
3. only by leader
4. independent with sharding 


2) Best trading algorithm to build on top [1-3]:
1. MacroIn 
2. NCRSI with macro
3. CRSI with macro


```

**Comment by adenysenko** [comment]
```
Context: Mastercard tech stuff

1) Best way to enhance "consent history" 
data with data from "consent document" [1-2]
1. adhoc query
2. async job

Context: Lexa starts studying programming.

2) Best laptop for Lexa [1-5]:
1. lenovo-legion-5-15
2. samsung-galaxy-book-flex-alpha
3. dell-inspiron-15-7000
4. msi-15-6-laptop
5. asus-rog-strix-g15


3) Check if it worth for me to subscribe for 
"Technical Analysis of STOCKS & COMMODITIES" magazine.

```

**Comment by adenysenko** [comment]
Context: Mastercard tech stuff.
We have async consent loading discussions.

1) Best things to do [1-2] 
1. insist on my design approach
2. follow with Vasily idea

**Comment by adenysenko** [comment]
```
1) Best architect to hire for Mastercard [1-2]:
1. Hemake Sannasgla 
2. Praveen Manthena

```

**Comment by adenysenko** [comment]
```
# Revolution/Aspiration - buy all ###

- bright
    - NVDA - earnings on 19
    - AAPL
    - TSLA
    - AMD - too late 

- # review later
    - EDUC
    - OSTK
    - SE 
    - SPOT
    - NVDA - divedent

- good but later
    - MSFT
    - AMD
    - AMZN - wait for SQ to finish
    - AAPL - wait for pullback? 

- PZZA
    - conf: sq, stacked emas, 
    - Buterfly SEPT 18 = 100/105/110
        - cost: $73->427
        - legs: 100-109.27

- AMZN - blue
    - 3345 ## ATH
    - 3148 - now
    - 3472 - 1.2
    - conf: early sq 11aug
        - 1 W - blue momentum
    - Buterfly SEPT 18 = 3320/3340/3360 # GRACE/EclipseLight
        - cost $43/1957=45
        - legs: 3320/3359
    - Buterfly SEPT 18 = 3340/3360/3380 #Conflict/Empowering
        - cost $60/1940
        - legs: 3420/3459

- # MA - bright
    - conf: SQ 3D, 10x 1D
        - 1W - momentum + sq
        - negativetove: KC1 
    - 326 - now
    - 347 - ATH
        - 18 SEPT
        - ATR=1.8% oer day
    - Buterfly SEPT 18 = 330/340/350 #
        - legs: 331-348
        - $172->827=4.8
    - # Buterfly SEPT 18 = 335/340/345 #CALDON                      #65 @@@@ ###
        - legs: 332-348
        - $65->435=6.69

- # ROKU - blue
    - ATH 170
    - price: 146
        - 0.618=160
    - Buterfly SEPT 18 = 150/155/160 # Breakthrough/Following       #28
        - cost 472/28 = 16 #####
    - Buterfly SEPT 18 = 165/170/175 #Attraction/CriticalMass
        - cost 465/35 = 12

- # PTON
    - ATH 73
    - now 64
    - Buterfly SEPT 18 = 65/70/75   #Aspiration/The End In Sight
        - cost 432/68 = ?
        - 65.68/74.32
    - Buterfly SEPT 18 = 70/75/80   #Creative Activity/Following    #32
        - cost 468/32 = ?
        - 70.32/79.68

- # CLX
    - ATH 240
    - now=224
    - 0.68=233
    - Buterfly SEPT 18 = 230/240/250 #CreativeActivity              #145
        - $145/885 
        - 231/248

- # AKAM -?
    - ath 116
    - 108
    - Buterfly SEPT 18 = 110/115/120 #DangerouslyDeep/Decrease
        - $95/405
        - 111/119
     - Buterfly SEPT 18 = 105/110/115 # Lying Low/Enthusiasm        $68
        - $68/432
        - 105/114

- QLYS 
    - ath 125
    - 102
    - 618 = 115
        - Buterfly SEPT 18 = 110/115/120 #Dangerously Deep
        - 30/470
        - 110/119

- STMP - sideways?

- # PYPL
    - ath 203
    - 191
    - Buterfly SEPT 18 =  190/200/210 # Durability/Breakthrough
        - 189/810
        - 192/208
      - Buterfly SEPT 18 =  200/210/220 #Breakthrough/Revolution        $100
        - 100/900
        - 201/219

- SHOP - blue
    - no, but W pullback into 8ema, maybe bounc from 34 EMA D
    - Buterfly SEPT 18 = 1045/1050/1055 # Aspiration/Splitting Apart ###
        - cost 575/72 = 7.9 #####
        - 
- # SQ
    - ath 160
    - 141
    - conf: no SQ, momentum D is decreasing, momentum divergence ###
    - Buterfly SEPT 18 = 145/150/155  #SplittingApart/Contemplation
        - 0.618 = 150
        - legs: 145/154
        - $40/460
    - Buterfly SEPT 18 =  155/160/165 #Recharging Power/Grace
        - legs: 155/164
        - $36/464

- # GOOGL
    - ath 1587, 1.2=1620, 0.78=1550
    - 1504
    - conf: sq, earlyIn, 
    - Buterfly SEPT 18 =  1540/1550/1560 (0.78) # Estrangement/Inexperience
        - $87/1090
    - Buterfly SEPT 18 =  1590/1600/1610 (1.2) #Attraction/Bonding
        - $125/875=7
        - 1591/1608

```

**Comment by adenysenko** [comment]
https://macro-ops.com/why-yields-are-about-to-rise-dirty-dozen/

https://macro-ops.com/monday-dirty-dozen-chart-pack-17/

https://macro-ops.com/monday-dirty-dozen-chart-pack-15/


https://macro-ops.com/monday-dirty-dozen-chart-pack-17/

**Comment by adenysenko** [comment]
```

1. Stoit li torgovat po alertam Raghee
2. Delat svoi short term future algos

mozhet vziat paru HL stocks i na nih zapuskat algos: macro, connors, ...., fire
proshe navernoe "macro-plays" - ih malo "set it and forget it"

NCRSI = 3
CRSI = 4
CTRIN = 5
C77 = 11
MacroIn = 13
YTX = 32

```

**Comment by adenysenko** [comment]
```
1) Best day for a trip [1-3]
1. 3
2. 4
3. 5

2) Mike is going to the sea for long weekend.
Is it worth for me and Lexa to join him?

```

**Comment by adenysenko** [comment]
Context: Mastercard tech stuff

1) best way to implement national ID support [1-3]
1. put extra config entries into shared "address-config" library
2. create a new shared "national-id-config" library
3. create a configuration section in consumer service

**Comment by adenysenko** [comment]
```
Hi Nahid,

Long time no hear, how is it going?
Just wanted to check how is it going generally with Mastercard and Main-St in general.

**Mastercard**
    What is about remote working? 
    Are they hiring?
    I think may be the rate is lowered so new resources may need to wait?

**Main Street**
    How is "main street" business? How is situation different from Wall st?
    Are you clients still hiring?
    Opening new projects?

Kakim obrazom sprosit est li interest?

```

**Comment by adenysenko** [comment]
- https://www.rightlinetrading.com/dr-keiths-seasonal-trade-alert-service/

2) Best trading course to get from rightlinetrading [1-3]:
	6	1. Seasonal Forex Day Trade Mastery Class
		2. Seasonal Futures Swing Trading Master Class
#	9	3. Both Courses

3) Best trade recommendation service  to get from rightlinetrading [1-6]:
#	9	1. Seasonal Forex Alert Service
#	9	2. Seasonal Futures  Alert

**Comment by adenysenko** [comment]
- https://newjersey.craigslist.org/sub/d/jersey-city-sublet-available-1br-fully/7191408350.html
- https://newjersey.craigslist.org/sub/d/jersey-city-1bed-1bath-apt-at-grove/7187122915.html old

**Comment by adenysenko** [comment]
**inflation protected bonds**
- https://etfdb.com/etfdb-category/inflation-protected-bonds/
- https://www.ivoletf.com/
- https://www.ivoletf.com/wp-content/uploads/2020/07/IVOL-Overview.pdf

**Comment by adenysenko** [comment]
# MODELS

MACRO-VXX
========================
Buy=false
Sell=true
Open: RUT/RSP
Close: RUT/RSP      RUT/QQEW - better ###
Close2: VXX

Exit Late = true
Exit Raf = false
Exit 80 = true

RSI Direction ONLY = true
VwapDerirection Only = false



MACRO-QQQ
========================
Open=HYG/SHY
Close=RUT/RSP
Close2=VIX3M/VIX
VWap deirection = true

Exit Late = true
Exit Raf = false
Exit 80 = true

VwapDerirection Only = true



VTR ####
========================

SEMA-BUY

QQQ
DIA
APPL
AMZN

SEMA-SELL

USO
XLE
TLT
HYG
APPL
EWZ

RSA-CROSS-BUY
QQQ
SPY
DIA
IWM
GLD
SLV
TLT
HYG
APPL
AMZN
GOOG
GS


RSA-CROSS-SELL

USO
XLE
HYG
IBM

**Comment by adenysenko** [comment]
```
278 ### last gold

// SELL VIX
VOL_VIX inversed on SPY (VIX>18 limit)
    - 18 trades 72% 8.33     100.63/4.98=20

//SELL VIX
VOL_VIX inversed on SPY (NO VIX>18 limit)
    - 50 trades 60% 3.62     119.24/7.74=15

MACRO_BUY on SPY
    - 55 traded 67% 3.54    138.56/11.52=12

```

**Comment by adenysenko** [comment]
# Jack's preference of Henry's Market Internals Grid: 
http://tos.mx/7L4gSKt

# Here is the TOS share link:
https://tos.mx/UejL2Ip

**Comment by adenysenko** [comment]
```
Context: Lexa will have another court meeting next Friday.
We a bit late with hiring detectives to dig info on her husband Adam but let's see.

1) Lets compare detectives [1-2]
1. "Bill Staton" team
2. Richie Cohen

2) Who will be the best option to look After this weekend [1-2]
1. "Bill Staton" team
2. Richie Cohen

3) Best period to look after Adam [1-3]
1. Saturday morning 9:30-11:00am with Sofie
2. Saturday evening near his house
3. Sunday afternoon  3:30-6:30pm with Sofie


```

**Comment by adenysenko** [comment]
```
1) General reading for the next week

2) Card and mantra for the next week

3) Best course/system to buy to make money [1-5]
1. Ultimate Investors Playbook
2. Systems Mastery Course
3. Trading Lab 2 months membership
4. System Building Masterclass from ScottPhillips
5. Prison Paycheck System from ScottPhillips

```

**Comment by adenysenko** [comment]
Context: PollinateTrading have a deal now for their membership 1 month for 2. 
I talk to their boss Chris, 
he may give me the same deal later in Jan/Feb but not sure.

1) Not sure should I do it now or later. Lets get a draw.

**Comment by adenysenko** [comment]
**1) Which day "S&P 500 CBOE Volatility Index (VIX)" close over 20?**
https://www.tradingview.com/chart/ff2eQvOc/
![image](https://user-images.githubusercontent.com/519787/129464148-681f57c0-86ef-4d08-be24-3357c0096408.png)


**2) Only if we have days over 20, for that days lets check if it goes over 25.**
```
	Leelahel yes has chances in days
	Leelahel like two to four days
	Leelahel after rising to rise higher
	Leelahel keep a close eye
	3 looks like a good chance
```

**3) Which day "CBOE VIX Volatility Index (VVIX)" close over 117?**
https://www.tradingview.com/chart/twDb93no/
![image](https://user-images.githubusercontent.com/519787/129464151-a6d8c799-ed60-4e46-adbb-72cc5a9be36d.png)

**Comment by adenysenko** [comment]
# PREV WEEK

**1) Which day "S&P 500 CBOE Volatility Index (VIX)" close over 20?**
https://www.tradingview.com/chart/ff2eQvOc/

![image](https://user-images.githubusercontent.com/519787/131041639-31955fdc-f6bd-4b7a-b954-6aa16721e59c.png)


**2) Which day "CBOE VIX Volatility Index (VVIX)" close over 120?**
https://www.tradingview.com/chart/twDb93no/

![image](https://user-images.githubusercontent.com/519787/131041517-03b905f8-e4e5-4529-9ca9-1a1f53cc8c8e.png)



# THIS WEEK

**1) Which day "S&P 500 CBOE Volatility Index (VIX)" close over 20?**
https://www.tradingview.com/chart/ff2eQvOc/
![image](https://user-images.githubusercontent.com/519787/132081067-d83004fe-6027-44b9-b200-9460fe94b02c.png)

**2) Which day "CBOE VIX Volatility Index (VVIX)" close over 120?**
https://www.tradingview.com/chart/twDb93no/
![image](https://user-images.githubusercontent.com/519787/132081081-cb30cffe-8154-4c96-bd81-b31bbc9ac7c6.png)



**3) Which direction "Invesco DB USD Bullish Fund ETF" goes till 4 September [1-3]:**
https://www.tradingview.com/chart/KB9rMSjp/
![image](https://user-images.githubusercontent.com/519787/132081171-0d39350e-43a2-4e42-995a-dbdf94cb2d91.png)

**4) Which direction "iShares 20+ Year Treasury Bond ETF" goes till 4 September [1-3]:**
https://www.tradingview.com/chart/PllRxe3y/
![image](https://user-images.githubusercontent.com/519787/132081175-829714aa-dfef-45fb-ae77-6575fd56a4f4.png)


# PREV WEEK

**1) Which day "S&P 500 CBOE Volatility Index (VIX)" close over 20?**
https://www.tradingview.com/chart/ff2eQvOc/

![image](https://user-images.githubusercontent.com/519787/130285991-7d00ef66-ddf2-4757-8f0c-346c2be52a5f.png)


**2) Which day "CBOE VIX Volatility Index (VVIX)" close over 120?**
https://www.tradingview.com/chart/twDb93no/


![image](https://user-images.githubusercontent.com/519787/130286158-8c59dee9-f75e-4449-8ca4-da27debc913a.png)
